extensions/wikia/Wall/WallThread.class.php

```diff
@@ -1,6 +1,8 @@
 <?php
 
 class WallThread {
+	const DATABASE_LIMIT = 500;
```

Comment:
I'm still open to suggestions for a better name for this constant. I'm not sure if there's a standard convention for naming constants at Wikia.